src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  width: 100%;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+  margin-top: 10px;
+}
+
+.list {
+  display: flex;
```

Comment:
You can use justify-content property as you used before, so that your don't need to hardcode paddings between elements. Probably that's the reason of not passing tests.